x-pack/test/detection_engine_api_integration/basic/tests/create_rules.ts

```diff
@@ -46,11 +47,13 @@ export default ({ getService }: FtrProviderContext) => {
     describe('creating rules', () => {
       beforeEach(async () => {
         await createSignalsIndex(supertest);
+        await esArchiver.load('auditbeat/hosts');
```

Comment:
For posterity: these indices were added so that the rule would not fail on the initial permissions check due to a lack of indices.